conference_101719.tex

```diff
@@ -0,0 +1,200 @@
+This is because the MAC addresses can be duplicated except those on the same path.
+Since the proposed scheme is independent of the real network environment, it is useful for overcoming the shortcoming of the previous scheme.
+In addition to that, the proposed scheme is not affected by a spoofed MAC address because it focuses on the only legitimate MAC address never spoofed.
+
+\subsection{Algorithm}
+In this subsection, the algorithm for detection based on searching the MAC address of LAP on the same path is explained.
+From this subsection, let \newcommand{\tarAP}{A_{target}}$\tarAP$ and \newcommand{\inputAP}{A_{input}}$\inputAP$ denote a targeted AP and input AP.
+In addition, a MAC address and SSID of $A_x$ are represented as $M_{A_x}$ and $S_{A_x}$, respectively.
+\newcommand{\tarMAC}{M_{A_{target}}} \newcommand{\tarSSID}{S_{A_{target}}} \newcommand{\userMAC}{M_{user\_device}}
+
+\subsubsection{Preparation for the detection}
+MAC addresses and SSIDs of APs are collected through beacon frames which exist in the communication range of a user device in preparation for the detection.
+Among them, APs whose SSIDs are identical to that of the targeted AP, namely \newcommand{\SSID}{S_{A_{target}}}$\SSID$, are extracted and represented as $AP_{all}=\{A_{input}|0\le input \le n_{all}\}$, where $n_{all}$ .
+
+\subsubsection{Flowchart of Our System}
+The algorithm consists of four procedures as shown in \ref{fig:flowchart}, which are 1) Comparison with $\tarMAC$, 2)Setting $\userMAC$ to \newcommand{\inputMAC}{M_{A_{input}}}$\inputMAC$ , 3)ARP request from a user device to a gateway and 4)Observation of ARP reply from the gateway to the user device.
+Note that they are repeatedly conducted for every MAC address in $APP_{all}$ unless a RAP is detected.
+
+\subsubsection{Comparison with $\tarMAC$}
+In the first phase, our system compares the $\inputMAC$ with the $\tarMAC$.
+If they are the same MAC address, namely $\inputMAC = \tarMAC$, it easily judges the either of them is RAP due to the cloned address.
+Thus, using the network should be avoided whether it is on the same path or not because the risk of being attacked is extremely high. 
+However, in the case where a RAP clones the MAC address which the LAP far from the user device has or it does not clone, our system cannot detect only by checking the accordance of MAC addresses.
+Thus, the detection process goes to the next phase.
+
+\subsubsection{Setting $\userMAC$ to $\inputMAC$}
+In this phase, in order to reveal the existence of LAP which is connected on the other side of the targeted AP if the user device is attacked, $\userMAC$ is set to $\inputMAC$ in preparation for the next procedure.
+
+\subsubsection{ARP Request from the User Device to the Gateway}
+After the MAC address is set to $\inputMAC$, the connection between the user device and $\tarAP$ is once lost since the gateway become unable to use original MAC address as a destination for the packets.
```

Comment:
```suggestion
After $\userMAC$ is set to $\inputMAC$, the connection between the user device and $\tarAP$ is once lost since the gateway become unable to use the original MAC address as a destination for the packets.
```